Companions/Luna/TutoringDialogues.cs

```diff
@@ -104,7 +104,7 @@ private static void AboutLeadingTerraGuardians()
             {
                 "*Depending on how much of a friend you are with someone, they may agree to follow you on your adventures. Some of them may not even mind joining your adventures at all, regardless of whether they know you or not.*",
                 "*You can also give them orders during your adventures, so they can take specific actions that you may find the need.*",
-                "*Pay attention to your group size. A companion may feel uncomfortable joining you if there are too many people in your group.*"
+                "*Pay attention to your group size. A companion may feel uncomfortable joining you if there are too many people in your group. Currently, you can 3 members in your group at once.*"
```

Comment:
I will need to change the number, to instead take the actual number of companions you can have following, so there is no need to keep updating this once the cap is increased.

Reply:
Good Idea, not sure how to implement that one myself though